### ISSUE TITLE
Screen readers are not announcing configuration condition labels for StringProperty text boxes in project property pages

### ISSUE THREAD

**Issue by japarson** [dotnet/project-system#7999]
## Visual Studio Version
Microsoft Visual Studio Enterprise 2022 Preview
Version 17.2.0 Preview 1.0

## Summary
Screen readers are not announcing configuration condition labels (e.g. Debug/Release) for `StringProperty` text boxes in the project property pages.
- [Bug 1504390](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1504390): A11y_Visual Studio Client_Android options_Uncompressed resources_Info and relationships: Narrator/NVDA is not announcing the "Debug" and "Release" label names for the edit fields.

https://user-images.githubusercontent.com/59936622/159581271-10532cd7-837a-42d8-9daa-424fa059023c.mp4

## Steps to Reproduce
1. Turn on Narrator/NVDA.
1. Create new Android project from power shell using `dotnet new android -n <project_name>`.
1. Open VS 2022 and tab till "Open a project or solution" option and press enter.
1. Tab till "File name" and select newly created iOS "csproj" file and press enter on "Open" button.
1. Tab till project in "Solution explorer" and press context key.
1. Tab till "Properties" menu item and press enter.
1. Tab till "Android Options" button and press enter.
1. Tab till "Debug" field and select "APK" option.
1. Tab till "Uncompressed resources" section present in "Packaging"
1. Tab till menu item and select "Vary value by configuration".
1. Tab till "Debug" and "Release" edit fields.
1. Check whether the screen reader is reading the label names for that edit field or not.

## Expected Behavior
Narrator/NVDA should announce the "Debug" and "Release" label names for the edit field.

## Actual Behavior
Narrator/NVDA is not announcing the "Debug" and "Release" label names for the edit field.

## User Impact
Screen reader users may not be able to know the information about the edit field if screen reader does not announce the label names.

[MAS 1.3.1 Info and Relationships](https://liquid.microsoft.com/Web/Views/View/410630#Zrex-3A-2F-2Fms-2Eaccessibility-2FRequirements-2F01-2E03-2E01-2F)

**Comment by japarson** [comment]
**Note:** This is also happening for `EnumProperty`s but not `BoolProperty`s.

**Comment by adamint** [comment]
Completed on ADO.
